src/flisp/julia_extensions.c

```diff
@@ -99,6 +99,9 @@ static int is_wc_cat_id_start(uint32_t wc, utf8proc_propval_t cat)
             (wc >= 0x2220 && wc <= 0x2222) || // ∠, ∡, ∢
             (wc >= 0x299b && wc <= 0x29af) || // ⦛, ⦜, ⦝, ⦞, ⦟, ⦠, ⦡, ⦢, ⦣, ⦤, ⦥, ⦦, ⦧, ⦨, ⦩, ⦪, ⦫, ⦬, ⦭, ⦮, ⦯
 
+	    // geometric shapes
+	    (wc >= 0x25a0 && wc <= 0x25ff) ||
+
```

Comment:
Aren't many of these already covered by the `cat == UTF8PROC_CATEGORY_SO` check?  It seems like the only ones in category Sm (which have to be special-cased) are U+25F8 to U+25FF.